index.bs

```diff
@@ -604,6 +613,10 @@ When this method is invoked, the user agent MUST execute the following algorithm
                 ::  A new {{ArrayBuffer}}, created using |global|'s [=%ArrayBuffer%=], containing the bytes of |clientDataJSON|.
                 :   {{AuthenticatorAttestationResponse/attestationObject}}
                 ::  |attestationObject|
+            :   {{ScopedCredential/clientExtensionResults}}
+            ::  A new {{AuthenticationExtensions}} object containing the [=extension identifier=] → [=client extension output=] entries
+                created by running each extension's [=client extension processing=] algorithm to create the [=client extension outputs=],
+                for each [=client extension=] in {{AuthenticatorResponse/clientDataJSON}}.clientExtensions.
```

Comment:
The authenticator extension outputs are returned in the authenticator data in CBOR map.